### PR TITLE
Updating FB installing instructions

### PR DESCRIPTION
**Pull request by vv-monsalve**
Following what was suggested in [this issue](https://github.com/fonttools/fontbakery/issues/4118), and after having a couple of Fails regarding freetype misssing.

### CODE REVIEW

File: gf-guide/tools.md
Comment (felipesanches) [inline gf-guide/tools.md:144]: `pip3 install -U fontbakery'[googlefonts]'` will install all dependencies needed to run checks from the Google Fonts profiles, which also includes freetype. Doing it this way will guarantee that, in the future, any additional dependencies will also be installed, as the Google Fonts profile continues evolving.

The same is valid for any other vendor-specific profile. There's one extra for each of them, so that installing is straightforward.

Comment (felipesanches) [reply gf-guide/tools.md:144]: oh! But what I just said is only valid for FontBakery version 0.9.0a1 or later (as of today it has not yet landed on a stable release, but only the most recent pre-releases of v0.9.0).

Comment (vv-monsalve) [reply gf-guide/tools.md:144]: So, do we wait for the stable version to land before updating the installation instructions? Or should we merge this here and then update it again when the new release happens? I would opt for the latter.

Comment (felipesanches) [reply gf-guide/tools.md:144]: yeah. Merge it now and update later.

Comment (vv-monsalve) [reply gf-guide/tools.md:144]: Okay, please approve it to unblock the merge.

**Review by RosaWagner — APPROVED** [review]

**Pull request merged**
